pennylane/ops/op_math/adjoint.py

```diff
@@ -205,51 +207,12 @@ class Adjoint(SymbolicOp):
 
     """
 
-    _operation_type = None  # type if base inherits from operation and not observable
-    _operation_observable_type = None  # type if base inherits from both operation and observable
-    _observable_type = None  # type if base inherits from observable and not operation
-
-    # pylint: disable=unused-argument
-    def __new__(cls, base=None, do_queue=None, id=None):
```

Comment:
Something I've been thinking about doing that just makes this a little bit easier.